CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+# [Please contribute](#please-contribute)
+
+You can really make a difference by:
+
+- [Making an issue](https://help.github.com/articles/creating-an-issue/). A well-described issue helps a lot. (Check the [known issues](https://github.com/search?q=user%3Avitabaks+is%3Aissue+state%3Aopen) first.)
+- [Making a pull request](https://docs.github.com/fr/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) when you find an error in the code.
+
+I will try to help and take every contribution seriously.
+
+This is a great opportunity for me to learn how you use the role and also an opportunity for you to get into the habit of contributing to open-source software.
+
+## [Step by step](#step-by-step)
+
+Here is how you can help, many steps are related to GitHub, not specifically to this project.
+
+### [1. Make an issue.](#1-make-an-issue)
+
+When you spot an issue, [create an issue](https://github.com/vitabaks/postgresql_cluster/issues).
+
+Making the issue helps me and others to find similar problems in the future.
+
+### [2. Fork the project.](#2-fork-the-project)
+
+On the top right side of [the repository on GitHub](https://github.com/vitabaks/postgresql_cluster), click `fork`. This copies everything to your GitHub namespace.
+
+### [3. Make the changes](#3-make-the-changes)
+
+In your GitHub namespace, make the required changes.
+
+I typically do that by cloning the repository (in your namespace) locally:
+
+```
+git clone git@github.com:YOURNAMESPACE/postgresql_cluster.git
+```
+
+Now you can start editing on your laptop.
+
+### [4. Test your changes](#4-optionally-test-your-changes)
+
+Install [make](https://www.gnu.org/software/make/) and [Python3.10](https://www.python.org/) with [venv](https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/) :
+
+```
+sudo apt install -y make python3.10 python3.10-venv
+python3 -m pip install --upgrade pip
+python3 -m pip install virtualenv
+```
+
+You also need [docker](https://docs.docker.com/engine/install/ubuntu/) installed.
+
+And run `make`. You will see help from Makefile
+
+```
+make
+```
+
+Run `make reinitialization` or `make bootstrap` for initializing the virtualenv and installing all dependencies.
+
+Then, run the test with `make molecule-test` or `make molecule-converge` for only the converge sequence.
+
+If you want to test a specific distribution, set `distro`, optionally `tag`, and `namespace` :
+
+```
+IMAGE_DISTRO=rockylinux9 IMAGE_TAG=latest IMAGE_NAMESPACE=geerlingguy make molecule-converge
```

Comment:
```
gitpod@thomassanso-postgresqlc-9nwq8iqng2j:/workspace/postgresql_cluster$ IMAGE_DISTRO=rockylinux9 IMAGE_TAG=latest IMAGE_NAMESPACE=geerlingguy make molecule-converge
/bin/bash: line 1: .venv/bin/activate: No such file or directory
Traceback (most recent call last):
  File "/usr/local/bin/molecule", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/click/decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/gitpod/.local/lib/python3.10/site-packages/molecule/command/converge.py", line 59, in converge
    base.execute_cmdline_scenarios(scenario_name, args, command_args, ansible_args)
  File "/home/gitpod/.local/lib/python3.10/site-packages/molecule/command/base.py", line 94, in execute_cmdline_scenarios
    get_configs(args, command_args, ansible_args, glob_str),
  File "/home/gitpod/.local/lib/python3.10/site-packages/molecule/command/base.py", line 185, in get_configs
    configs = [
  File "/home/gitpod/.local/lib/python3.10/site-packages/molecule/command/base.py", line 186, in <listcomp>
    config.Config(
  File "/home/gitpod/.local/lib/python3.10/site-packages/molecule/config.py", line 64, in __call__
    obj.after_init()
  File "/home/gitpod/.local/lib/python3.10/site-packages/molecule/config.py", line 118, in after_init
    self._validate()
  File "/home/gitpod/.local/lib/python3.10/site-packages/molecule/config.py", line 456, in _validate
    errors = schema_v3.validate(self.config)
  File "/home/gitpod/.local/lib/python3.10/site-packages/molecule/model/schema_v3.py", line 61, in validate
    with open(schema_file, encoding="utf-8") as f:
FileNotFoundError: [Errno 2] No such file or directory: '/usr/local/lib/python3.10/dist-packages/molecule_docker/driver.json'
make: *** [.config/make/molecule.mak:14: molecule-converge] Error 1
```

Reply:
The error encountered is expected, as the package python3.10-venv couldn't be installed, which prevents the creation of the Python virtual environment. This issue will be resolved once the missing command is added earlier in the process, allowing for the proper installation of python3.10-venv and successful creation of the virtual environment.

Reply:
Done and tested on gitpod (working)